.github/workflows/docs.yml

```
@@ -0,0 +1,43 @@
+# LLVM Documentation CI
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: "Test documentation build"
+
```

```suggestion

permissions:
  contents: read
  
```

This is standard for all our workflow files.

---

Good point. Didn't realize that Github by default doesn't give a minimal permissions set.